tests/ssl-opt.sh

```diff
@@ -3966,6 +3980,7 @@ if [ -n "$MAX_IM_CA_CONFIG" ] && [ "$MAX_IM_CA_CONFIG" -ne "$MAX_IM_CA" ]; then
 fi
 
 requires_full_size_output_buffer
+requires_config_value_at_least "MBEDTLS_SSL_MAX_CONTENT_LEN" 4096
```

Comment:
This is the wrong check - we should check that `MBEDTLS_X509_MAX_INTERMEDIATE_CA` is `8` here and below, and remove the corresponding `if ... exit fi` block above.